flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniClusterConfiguration.java

```diff
@@ -35,6 +36,8 @@
 /** Configuration object for the {@link MiniCluster}. */
 public class MiniClusterConfiguration {
 
+    static final int DEFAULT_IO_POOL_SIZE = 8;
```

Comment:
Since the `Dispatcher`/`JobMaster` and the `TaskExecutors` don't share the same `ioExecutor` I would even say that we can set the default pool size to `4`.